code/lib/core-common/src/utils/get-storybook-info.ts

```diff
@@ -98,7 +98,7 @@ const findConfigFile = (prefix: string, configDir: string) => {
 
 const getConfigInfo = (packageJson: PackageJson, configDir?: string) => {
   let storybookConfigDir = configDir ?? '.storybook';
-  const storybookScript = packageJson.scripts?.['storybook'];
```

Comment:
Is this change intended?

Reply:
Yes, but it shouldn't have been included in a merge, sorry for that.
If you prefer the original notation I can revert it.